ci-operator/config/openshift/console-plugin-template/openshift-console-plugin-template-main.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  nodejs-10:
+    name: nodejs-10-rhel7
+    namespace: openshift
+    tag: latest
+build_root:
+  from_repository: true
```

Comment:
We can remove this since we don't need any yaml on the template side to build the image